src/compiler/compiler.rs

```diff
@@ -955,7 +955,7 @@ where
 
         match rustc_vv {
             Ok(rustc_verbose_version) => {
-                debug!("Found rustc");
+                debug!("Found rustc at path: {rustc_executable:?}");
```

Comment:
Might want to log the unwrapped executable instead.

Reply:
Addressed! Current logging:

```shell
[2023-01-11T09:01:05Z DEBUG sccache::compiler::compiler] Using rustc at path: "/home/xuanwo/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/rustc"
```